docs/en/03_Upgrading/03_Migrating_off_CWP.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Migrating off CWP
+summary: Migrate your project off the Common Web Platform v2
+---
+
+# Migrate your project off the Common Web Platform v2
+
+Until November 2021, Silverstripe Ltd maintained a specilasied version of the Silverstripe CMS project for the New Zealand Goverment and related New Zealand public agencies. This version was called _Common Web Platform_ (CWP for short).
+
+The CWP CMS release has been discontinued. CWP project who wish to continue to receive update most migrate their code base off the latest CWP release back to the regular Silverstripe CMS release.
+
+## Steps to migcate away from CWP
+
+Before you began this process you must identify which version of Silverstripe CMS Recipe you want to upgrade to. You can upgrade to [Silverstripe CMS Recipe 4.9.0](/changelogs/4.9.0/) or greater.
+
+[Review the changelog list](/changelogs/) to choose the Silverstripe CMS Recipe version you wish to upgrade to. Read the changelog carefully. It may contain additional release specific upgrade guidance beyond the steps in this guide.
+
```

Comment:
Do projects need to be using [silverstripe/recipe-plugin](https://github.com/silverstripe/recipe-plugin) to run the below commands? If so, there's probably a need for a 'prerequisites section?

Reply:
`recipe-plugin` is a requirement of all recipes. You can't have a recipe without having it.

Reply:
Alright. Resolving.